Add "Get farmOS, "Develop farmOS", and "Follow farmOS" headers and links to homepage

mstenta opened the pull request:
This add a bunch of links to the homepage, based on feedback we've heard from the community. It also adds 3 `<h2>` headers to organize everything:

- Get farmOS
- Develop farmOS
- Follow farmOS

wotnak commented:
Looks good, although I'm not sure if the repositories list is really needed on the homepage. It pushes the 'Follow farmOS' section down, and Github org with all repos is already linked right above it. Maybe a better place to list important git repos would be in the [Github org README](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#organization-profile-readmes).

mstenta commented:
For what it's worth, part of the motivation for adding the repositories was related to: https://github.com/farmOS/farmOS.org/pull/22#issuecomment-1892012536

mstenta commented:
Forgot about this PR!

Looking at it with fresh eyes, I agree with @wotnak's comment, so I removed the individual repository links.

Rebased and force pushed. Ready for review again. :-)

wotnak commented on src/content/index.md:
```diff
@@ -19,6 +19,28 @@ You can contribute to the project by [making a donation] or [contributing] in
+## Develop farmOS
+
+To join the farmOS community development efforts, read the
+[Contributing to farmOS] overview, and join the community forum, chat, or calls.
```
there's no url set for this link
![image](https://github.com/user-attachments/assets/59bd206c-0cec-4fad-82f2-1daeccf72731)

mstenta replied:
Thanks @wotnak! Fixed and force pushed.
